README.md

```diff
@@ -335,7 +334,7 @@ The associated [`SafeModule` class](torchrl/modules/tensordict_module/common.py)
     ```python
     from torchrl.objectives import DQNLoss
     loss_module = DQNLoss(value_network=value_network, gamma=0.99)
-    tensordict = replay_buffer.sample(batch_size)
+    tensordict = replay_buffer.sample(batch_size)[0]
```

Comment:
What's your view on this?
I feel that the `[0]` index is bothersome. With `TensorDictReplayBuffer` we read the second output from `sample` (`info`) and write the content in the tensordict.
Maybe we should just have a `def sample(self, ..., return_info=False)` flag that can be turned on when needed.
Wdyt?

Reply:
I also don't like the `[0]` index. However, the version of `TensorDictReplayBuffer` proposed in `rb_prototype.py` also returns 2 values from `sample()`. While migrating I decided to keep this logic. If we change it, we break consistency with parent class `ReplayBuffer`. I don't know if there are cases when users would like to use both `ReplayBuffer` and `TensorDictReplayBuffer` interchangeably in a single piece of code. If this is not the case, we can stop returning info (or return it only if the flag is set).

Reply:
Ok, I see two ways of going about this
1. branch out your branch and have a `sample()` function with a single output, merge it after review in your branch and ship the whole thing. Advantage: we don't bc-break nightly twice
2. ship this PR as it is and solve it in a second time.
Any pref?

Reply:
> I don't know if there are cases when users would like to use both ReplayBuffer and TensorDictReplayBuffer 

It's better if they're consistent, even if they're not interchangeable

Reply:
I think that the last approach (1 value by default in base and derived classes) is best.
If you agree, I'll make a new PR with this change.

Reply:
yeah, one value by default unless return_info = True
Thanks for taking care of this!

Reply:
I've updated this PR with the required changes.